sync2jira/downstream_issue.py

```diff
@@ -1071,7 +1140,20 @@ def _update_github_project_fields(
                     )
                 continue
             try:
-                jirafieldname = default_jira_fields["storypoints"]
+                field_identifier = default_jira_fields.get("storypoints")
+                if not field_identifier:
+                    log.error(
+                        "Configuration error: Missing 'storypoints' in `default_jira_fields`"
+                    )
+                    continue
+                # Resolve the field identifier to an ID
+                jirafieldname = _resolve_field_identifier(client, field_identifier)
+                if not jirafieldname:
+                    log.error(
+                        f"Could not resolve custom field '{field_identifier}' to an ID, skipping"
+                    )
+                    continue
+
                 log.info(f"Jira issue story point field name is:  '{jirafieldname}'")
```

Comment:
While you're in here, consider changing this log message from `INFO`-level to `DEBUG`.